### PR TITLE
crud for facilities; csv upload for accounts+facilities

### PR DESCRIPTION
**Pull request by dhorskyi**
added utility CRUD for facilities; refactored CSV upload for accounts+facilities
adding tests; ordering jira tasks underway

### CODE REVIEW

**Comment by arnon-flwr** [comment]
:star: PR built and available in a preview environment **flwr-technologies-scribe-pr-25**

**Comment by arnon-flwr** [comment]
:star: PR built and available in a preview environment **flwr-technologies-scribe-pr-25**

**Comment by arnon-flwr** [comment]
:star: PR built and available in a preview environment **flwr-technologies-scribe-pr-25**

File: src/main/scala/com/flwr/scribe/endpoints/accounts/AccountLogics.scala [hunk 1]
```diff
@@ -83,58 +83,61 @@ class LogicAndRoutes(repository: AccountRepository)(implicit ec: ExecutionContex
...
-      else
-        Left(
-          errors
-            .map { failMessageAndSource =>
-              s"the row '${failMessageAndSource._2}' failed because of: '${failMessageAndSource._1}'"
-            }
-            .mkString("\n")
-        )
-    }
+    // ===
```
Comment (Arnonrgo) [inline src/main/scala/com/flwr/scribe/endpoints/accounts/AccountLogics.scala]: if it isn't just for quick testing - remove the code

File: src/main/scala/com/flwr/scribe/endpoints/accounts/AccountsEndpoints.scala [hunk 2]
```diff
@@ -107,15 +116,14 @@ trait AccountsEndpoints {
...
-    .description("Upload an accounts CSV (binary format")
-    .in("csvBin")
-    .in(binaryBody[Array[Byte]]
-      .example("legalName,firstName,lastName,roleTitle,emails,phoneNumbers".getBytes))
-    //todo return a list of failed rows
-  //    .out(stringBody(StandardCharsets.UTF_8))
+//  val uploadCsvBinaryEndpoint: Endpoint[Array[Byte], Seq[String], Unit, Nothing] =
+//  accountsBaseEndpoint
+//    .post
+//    .description("Upload an accounts CSV (binary format")
```
Comment (Arnonrgo) [inline src/main/scala/com/flwr/scribe/endpoints/accounts/AccountsEndpoints.scala]: same here - remove the commented code

File: src/main/scala/com/flwr/scribe/endpoints/accounts/AccountLogics.scala (hunk 1 above)
Comment (dhorskyi) [reply src/main/scala/com/flwr/scribe/endpoints/accounts/AccountLogics.scala]: Sure, will remove all not-needed comments

**Comment by arnon-flwr** [comment]
:star: PR built and available in a preview environment **flwr-technologies-scribe-pr-25**

**Pull request merged**